ci-operator/config/openshift-kni/eco-ci-cd/openshift-kni-eco-ci-cd-main__far-edge-vran-dast-4.16.yaml

```diff
@@ -0,0 +1,39 @@
+base_images:
+  base:
+    name: "4.16"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    name: ci
+    namespace: openshift
+    tag: golang-1.20
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.16"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: eco-ci-cd
+  cluster: build05
+  cron: 00 13 * * 1
+  steps:
+    env:
```

Comment:
Please remove one env as there are 2 in your config.

Reply:
done